interfaces/builtin/core-support.go

```diff
@@ -0,0 +1,130 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const coreSupportConnectedPlugAppArmor = `
+# Description: Can start/stop/restart existing services via the systemd dbus API.
+# This interface gives privileged access to all snap and system services.
```

Comment:
Since we also allow /etc/ssh/sshd_not_to_be_run, can you change this to:

```
# Description: Can start/stop/restart existing services via the systemd dbus API
# and enable/disable ssh. This interface gives privileged access to all snap and
# system services.
```